File: Leetcode/Day-3/Python/Find the duplicate number.py
from collections import Counter
class Solution(object):
    def findDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = Counter(nums)

        for key, value in count.items():
            if value >= 2:
                return key

        return -1

File: Leetcode/Day-3/java/find the duplicate number.java
import java.util.*;
class Solution {
    public int findDuplicate(int[] nums) {
        HashMap<Integer, Integer> map = new HashMap<>();
        for (int i: nums){
            map.put(i, map.getOrDefault(i, 0)+1);
        }

        for (int i = 0; i < nums.length; i++){
            if (map.get(nums[i]) >= 2){
                return nums[i];
            }
        }

        return -1;
    }
}
